src/FileSystem-Core-Tests/PathTest.class.st

```diff
@@ -4,6 +4,7 @@ SUnit tests for file system paths
 Class {
 	#name : 'PathTest',
 	#superclass : 'TestCase',
+	#type : 'variable',
```

Comment:
Is it necessary to change the layout of this test class?

Reply:
> Is it necessary to change the layout of this test class?

It is not.  I'm missed that because I did not add it to the code.  It was probably done by some action I preformed.  Thank you for catching it!